brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
@ericmj I don't want the burden of my changes to fall on anyone else. How can I test this so we all feel comfortable with the change? Is there a test-bed project that I can use that exercises `phoenix.js` and the excellent work that you did to get it well-tested in IE8?